data_summaries/AgentInstruct.json

```diff
@@ -0,0 +1,301 @@
+{
+    "AgentInstruct-alfworld": {
+        "Unique Dataset Identifier": "AgentInstruct-alfworld",
+        "Dataset Name": "ALFworld",
+        "Paper Title": "ALFWorld: Aligning Text and Embodied Environments for Interactive Learning",
+        "Dataset URL": "https://github.com/alfworld/alfworld/tree/master/alfworld/data",
+        "GitHub URL": "https://github.com/THUDM/AgentTuning",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/alfworld",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2010.03768",
+        "Semantic Scholar Corpus ID": 222208810,
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Instruction Following",
+            "Chain-of-Thought"
```

Comment:
it's a bit of a pain, but are there more categories that are clearly present from a quick manual inspection of the data?

Reply:
we could create a new category for „reasoning traces and task-specific actions“, particularly „task-specific actions“. This would provide a better description of the data instead of just having a „chain of thoughts“.

Reply:
based on the existing tasks, I think that it falls under the category of "Commonsense Reasoning" > "Decision Making" etc.  I have added it as an additional task. I would be happy to hear your thoughts on this, and whether there is a better fit for it